packages/react/.storybook/story-config.ts

```diff
@@ -0,0 +1,93 @@
+/**
+ * Copyright (c) 2020, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
```

Comment:
Shall we update this year?

Reply:
Fixed with https://github.com/wso2/oxygen-ui/commit/60d1b964cd88e1d9dc78c533b73150a895b4f2e6